priv/signs.json

```diff
@@ -9738,6 +9738,24 @@
             "direction_id": 0
           }
         ]
+      },
+      {
+        "sources": [
+          {
+            "stop_id": "185",
+            "route_id": "716",
+            "direction_id": 0
+          }
+        ]
+      },
+      {
+        "sources": [
+          {
+            "stop_id": "185",
+            "route_id": "716",
+            "direction_id": 1
```

Comment:
It looks like Mattapan is a terminal for this route, so we probably only expect to see predictions in one direction, not both.